lib/optimizely/config_manager/http_project_config_manager.rb

```diff
@@ -148,16 +148,18 @@ def fetch_datafile_config
     end
 
     def request_config
-      @logger.log(
-        Logger::DEBUG,
-        "Fetching datafile from #{@datafile_url}"
-      )
-      begin
-        headers = {}
-        headers['Content-Type'] = 'application/json'
-        headers['If-Modified-Since'] = @last_modified if @last_modified
-        headers['Authorization'] = "Bearer #{@access_token}" unless @access_token.nil?
+      @logger.log(Logger::DEBUG, "Fetching datafile from #{@datafile_url}")
+      headers = {}
+      headers['Content-Type'] = 'application/json'
+      headers['If-Modified-Since'] = @last_modified if @last_modified
+      headers['Authorization'] = "Bearer #{@access_token}" unless @access_token.nil?
```

Comment:
We probably shouldn't output auth tokens to the logs.

Reply:
You are right, that is exactly what i did. This is why i cleansed the headers before logging and replaced the value of `Authorization` header to `***` right below this line.